source/css/scss/vendor_override/foundation/_buttons.scss

```diff
@@ -86,17 +85,21 @@ button, a.button, .button {
 }
 
 button.secondary, .button.secondary {
-	border-color: $kiva-text-dark;
+	border-color: $charcoal;
 	border-width: 1px;
 	font-size: $secondary-button-font-sm;
+  	padding: rem-calc(16) rem-calc(32);
+  	box-shadow: 0 1px #333;
+  	border-radius: $button-border-radius;
 
 	&:visited {
 		color: $kiva-text-dark;
 	}
 
 	.no-touch &:hover {
-		border-color: $kiva-textlink-hover;
-		color: $kiva-textlink-hover;
+		border-color: $kiva-accent-blue;
+		color: $kiva-accent-blue;
+	  	box-shadow: 0 1px $kiva-accent-blue;
```

Comment:
since the border is the same color, this will read as just a double-width bottom border rather than a shadow

Reply:
Got verification from brandon that is how we would like it.